### ISSUE TITLE
Refine IP addressing choices

### ISSUE THREAD

**Issue by scop** [UpCloudLtd/upcloud-whmcs-module#48]
We should refine the IP address choices made available when creating a server.

- Consider not offering the option to create the Utility IP/interface.
- Consider removing the addressing option altogether, and always going with either public IPv4 only, or public IPv4+IPv6.

**Comment by thevilledev** [comment]
As discussed internally, public IPv4 only for now

**Comment by adityawbwce** [comment]
Open the lib/ConfigOptions.php file and remove the code between lines #40 to #45, which is as follows:
private function getNetworking()
```
{
    $nw['ipv4only'] = 'Public IPv4 Only';
     $nw['default'] = 'Public Default (IPv4, IPv6 & Utility)';
     return $nw;
 }
```
Also, remove line #81, which contains the following code within lib/ConfigOptions.php file: 
```$this->createIPFields($groupId, $this->getNetworking());```

Open the lib/vmManager.php file and find the line #69 and replace from
```$networking = $this->params['configoptions']['ipaddress'];```
to
```$networking = "ipv4only";```

**Comment by scop** [comment]
Fixed by #60
